reinforcement_learning/rl_unity_ray/Dockerfile

```diff
@@ -11,8 +11,6 @@ RUN pip install --upgrade \
     gym-unity \
     mlagents-envs
 
-RUN pip install sagemaker-containers --upgrade
```

Comment:
any reason to remove to this line?

Reply:
Thanks, this was part of the previous change. Removed.